tensorflow/compiler/xla/service/cpu/simple_orc_jit.cc

```diff
@@ -41,6 +40,19 @@ limitations under the License.
 #include "tensorflow/compiler/xla/types.h"
 #include "tensorflow/core/platform/logging.h"
 
+#ifdef _MSC_VER
```

Comment:
I'd prefer not cluttering up this file, and instead put these platform-specific (missing) functions in a separate file.  Let's see what @sanjoy thinks about that.

Reply:
> I'd prefer not cluttering up this file, and instead put these platform-specific (missing) functions in a separate file.

Agree.

Reply:
+1 to this.

Reply:
@sanjoy Any idea where should I put them though? I am still learning the structure of Tensorflow codebase.

Reply:
I'd create a `msvc_compatibility.cc/h` (or `windows_compatibility.cc/h`, if that's better) and put these declarations in there, guarded under `_MSC_VER`.

Reply:
Done.

Reply:
I'd avoid putting the `_MSC_VER` stuff here at all -- instead guard the entire contents of `windows_compatibility.cc/h` undef `_MSC_VER`.

Secondly, is `_MSC_VER` the right macro to use for this?  Will it work do the right thing if we build with clang on windows?